flow/IThreadPoolTest.actor.cpp

```diff
@@ -60,6 +72,41 @@ TEST_CASE("/flow/IThreadPool/NamedThread") {
 	return Void();
 }
 
+TEST_CASE("/flow/IThreadPool/ThreadReturnPromiseStream") {
```

Comment:
Could you write a test case for sending an error as well?

Reply:
Done.